Sources/Testing/SourceAttribution/Expression.swift

```diff
@@ -236,24 +280,19 @@ public struct __Expression: Sendable {
         }
       }
 
-      description = String(describingForTest: subject)
-      debugDescription = String(reflecting: subject)
-      typeInfo = TypeInfo(describingTypeOf: subject)
-      self.label = label
-
-      isCollection = switch mirror.displayStyle {
-      case .some(.collection),
-           .some(.dictionary),
-           .some(.set):
-        true
-      default:
-        false
-      }
-
       if shouldIncludeChildren && (!mirror.children.isEmpty || isCollection) {
-        self.children = mirror.children.map { child in
-          Self(_reflecting: child.value, label: child.label, seenObjects: &seenObjects)
+        var children: [Self] = []
+        for (index, child) in mirror.children.enumerated() {
+          if isCollection && index >= configuration.maximumValueReflectionCollectionCount {
+            isTruncated = true
+            let message = "(\(mirror.children.count - index) elements (out of \(mirror.children.count) total) omitted for brevity.)"
```

Comment:
```suggestion
            let message = "(\(mirror.children.count - index) of \(mirror.children.count) elements omitted for brevity.)"
```

To avoid the double parentheses. Unsure if the trailing period is appropriate.